### PR TITLE
removing provider label from tiers

### PR DESCRIPTION
**Pull request by tinakurian**
Removing provider label from namespace template tiers.

Jira story: https://issues.redhat.com/browse/CRT-495

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -117,8 +117,12 @@ github.com/cockroachdb/cockroach-go v0.0.0-20181001143604-e0a95dfd547c/go.mod h1
 github.com/codegangsta/negroni v1.0.0/go.mod h1:v0y3T5G7Y1UlFfyxFn/QLRU4a2EuNau2iZY63YTKWo0=
 github.com/codeready-toolchain/api v0.0.0-20200402215020-c7a5434db5fb h1:jKoTRIPS80hYsFlZs6fIYXcuvr7fbz1y28xbFuTj0Pc=
 github.com/codeready-toolchain/api v0.0.0-20200402215020-c7a5434db5fb/go.mod h1:w07CIyd2cCp1i38hdPrG4aZPbwISzMkxOmtCWE4EtdY=
+github.com/codeready-toolchain/api v0.0.0-20200403111848-efd1f59b2e49 h1:A1QJx5ZrTFnQVbBYuiS6bqnFcPy7xS3fIpzrmEvKbdQ=
```
Comment (alexeykazakov) [inline go.sum:120]: Can you please do `go mod tidy`.

Comment (tinakurian) [reply go.sum:120]: fixed here: 0d9fbb0160fb4294218fc741b85ec9dd4b337ca9

**Comment by MatousJobanek** [comment]
@tinakurian you forgot to remove it from the test templates: https://github.com/codeready-toolchain/host-operator/tree/master/test/templates/nstemplatetiers

**Pull request merged**